### PR TITLE
set couch-stale setting to update_after

### PR DESCRIPTION
**Pull request by sravfeyn**
@proteusvacuum 
FYI @snopoke

### CODE REVIEW

**Comment by snopoke** [comment]
:+1: just need to make sure this is overridden in the env vars files where it needs to be (prod and softlayers)

File: ansible/roles/commcarehq/templates/localsettings.py.j2 [hunk 1]
```diff
@@ -460,7 +460,8 @@ INDICATOR_NAMESPACES = {
         ('mvp_indicators', "MVP"),
     ),
 }
-COUCH_STALE_QUERY = 'ok'
+# should be 'ok' if couchdb version < 1.1.0, else 'update_after'
```
Comment (czue) [inline ansible/roles/commcarehq/templates/localsettings.py.j2]: nit, but it should also be 'ok' on any cloudant environment

Comment (snopoke) [inline ansible/roles/commcarehq/templates/localsettings.py.j2]: Maybe also say that it should be 'ok' for cloudant.

File: ansible/roles/commcarehq/templates/localsettings.py.j2 [hunk 2]
```diff
@@ -460,7 +460,8 @@ INDICATOR_NAMESPACES = {
...
+COUCH_STALE_QUERY = 'update_after'
```
Comment (czue) [inline ansible/roles/commcarehq/templates/localsettings.py.j2]: like @snopoke said - we don't want to change this for prod/india so should be overrideable and the default should probably remain 'ok'

Comment (czue) [inline ansible/roles/commcarehq/templates/localsettings.py.j2]: like @snopoke said - we don't want to change this for prod/india so should be overrideable and the default should probably remain 'ok'

**Comment by sravfeyn** [comment]
Makes sense, closing this

**Comment by sravfeyn** [comment]
Updated

**Comment by sravfeyn** [comment]
Bump. This is urgent, exports are dysfunctional on swiss without this

**Pull request merged**